Source/Core/Core/NetPlayServer.cpp

```diff
@@ -273,6 +277,8 @@ unsigned int NetPlayServer::OnConnect(ENetPeer* socket)
   Client player;
   player.pid = pid;
   player.socket = socket;
+  player.qos_session = nullptr;
```

Comment:
This isn't needed if `qos_session` is a `std::unique_ptr`, since they initialize to `nullptr` by default.